x/wasm-storage/types/tx.go

```diff
@@ -87,3 +87,15 @@ func (msg MsgInstantiateAndRegisterProxyContract) ValidateBasic() error {
 	}
 	return nil
 }
+
+func (m *MsgUpdateParams) ValidateBasic() error {
```

Comment:
maybe call it `Validate()` to avoid confusion? (since `ValidateBasic()` has been deprecated)